src/Device.ts

```diff
@@ -197,4 +197,6 @@ class Device {
     }
 }
 
-new Device().start();
+const networkInterface = getParameter("interface");
```

Comment:
Leaving outside the class allows for reuse of the Device class.  I'll put it inside if that's the preference, but it seems clearer to handle outside the encapsulation of the Device.  Device is the device, and the start function takes a network interface.  Keeping it outside maintains the integrity of the encapsulation.  Yeah?

Reply:
The Device class is currently not meaned to be "reusable" ... also see that all the other parameters are read in there. It is just a class in that CLI for the main functions. This will also all change soon when I come to the new High level API :-)